packages/react-components/src/icons/info-circle-yellow.tsx

```diff
@@ -0,0 +1,26 @@
+/* istanbul ignore file */
+
+const infoCircleYellow = (
```

Comment:
This should be added to the icons storybook. Surprised we don't already have this